card/mod/05_standard/set/type/search_type.rb

```diff
@@ -13,7 +13,9 @@ def item_names params={}
 end
 
 def item_type
-  query[:type]
+  if !query[:type].is_a?(Array) && !query[:type].is_a?(Hash)
```

Comment:
Use a guard clause instead of wrapping the code inside a conditional expression.